bpfman/src/main.rs

```diff
@@ -23,10 +26,18 @@ const BPFMAN_ENV_LOG_LEVEL: &str = "RUST_LOG";
 
 #[cfg(not(test))]
 lazy_static! {
-    pub static ref ROOT_DB: Db = Config::default()
-        .path(STDIR_DB)
-        .open()
-        .expect("Unable to open root database");
+    pub static ref ROOT_DB: Db = {
```

Comment:
This looks like a super simple fix and I think it will work, however I think we'd want to be able to easily configure the maximum number of retries and duration between retries

Reply:
It's definitely going to be a bit annoying to tune those parameters in some scenarios